src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2280,6 +2276,17 @@ Regular\ expression=Regular expression
 
 Error\ importing.\ See\ the\ error\ log\ for\ details.=Error importing. See the error log for details.
 
+Error\ from\ import\:\ %0=Error from import\: %0
+Error\ reading\ PDF\ content\:\ %0=Error reading PDF content\: %0
+Error\ reading\ XMP\ content\:\ %0=Error reading XMP content\: %0
+Importing\ bib\ entry=Importing bib entry
+Importing\ using\ XMP\ data...=Importing using XMP data...
+Importing\ using\ extracted\ PDF\ data=Importing using extracted PDF data
+No\ bibtex\ data\ found.\ Creating\ empty\ entry\ with\ file\ link=No bibtex data found. Creating empty entry with file link
```

Comment:
5 years ago, We had a huge debate how to write BibTeX. We decided for `BibTeX` (see https://github.com/koppor/jabref/issues/434). Could you please change it here, too?